tests/test_authz_authorizer.py

```diff
@@ -1,35 +1,29 @@
 # coding=utf-8
+from copy import deepcopy
 from datetime import datetime, timedelta
+from unittest.mock import patch, MagicMock, call
 
 import pytest
 
-from lbz.authz import Authorizer, ALL, ALLOW, DENY, LIMITED_ALLOW
+from lbz.authz.authorizer import Authorizer, ALL, ALLOW, DENY, LIMITED_ALLOW
 from lbz.exceptions import PermissionDenied, SecurityRiskWarning, Unauthorized
 from tests import SAMPLE_PRIVATE_KEY, EXPECTED_TOKEN
 
 
 # pylint: disable=too-many-public-methods
-class TestAuthorizer:
-    def setup_method(self):
+class TestAuthorizerSetupClass:
```

Comment:
You need to prepare better (more descriptive) names for the tests classes or just keep all tests in one as it was before 😉